firebase-vertexai/src/main/kotlin/com/google/firebase/vertexai/Chat.kt

```diff
@@ -70,9 +78,17 @@ public class Chat(
   }
 
   /**
-   * Generates a response from the backend with the provided text prompt.
+   * Sends a message using the existing history of this chat as context and the provided text
+   * prompt.
+   *
+   * If successful, the message and response will be added to the history. If unsuccessful, history
+   * will remain unchanged.
    *
-   * @param prompt The text to be converted into a single piece of [Content] to send to the model.
+   * This function should be called within a suspend context to properly manage concurrency.
+   *
+   * @param prompt The input(s) that, together with the history, will be given to the model as the
```

Comment:
"input(s)" makes sense with the `Content` methods, but seems a bit off when the method can only accept a single String or Bitmap. This is the case for a handful of similar messages.